src/Stomp/Network/Observer/ConnectionObserverCollection.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/*
+ * This file is part of the Stomp package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Stomp\Network\Observer;
+
+
+use Stomp\Transport\Frame;
+
+/**
+ * ConnectionObserverCollection a collection of connection observers.
+ *
+ * @package Stomp\Network\Observer
+ * @author Jens Radtke <swefl.oss@fin-sn.de>
+ */
+class ConnectionObserverCollection implements ConnectionObserver
+{
+    /**
+     * @var ConnectionObserver[]
+     */
+    private $observer = [];
+
+    /**
+     * Adds new observers to the collection.
+     *
+     * @param ConnectionObserver $observer
+     * @return ConnectionObserverCollection
```

Comment:
I used to replace `$this` with the actual return type since some time now. I also see that at symfony  code - I would just like to change it to something like `@return ConnectionObserverCollection this collection` - what do you think about that?